ui_src/config/nginx.conf

```diff
@@ -0,0 +1,14 @@
+upstream api_server {
```

Comment:
This file is not needed as it will be transferred to the ansible repository. It can be deleted from here.